pages/linux/readpe.md

```diff
@@ -0,0 +1,28 @@
+# readpe
+
+> Displays information about PE files.
+> More information: <https://manpages.debian.org/testing/pev/readpe.1.en.html>.
+
+- Display all information about the PE file:
+
+`readpe {{path/to/binary.exe}}`
+
+- Display all the headers present in the PE file:
+
+`readpe --all-headers {{path/to/binary.exe}}`
+
+- Display all the sections present in the PE file:
+
+`readpe --all-sections {{path/to/binary.exe}}`
+
+- Display a specific header from the PE file:
+
+`readpe --header {{dos|coff|optional}} {{path/to/binary.exe}}`
+
+- Display imported functions:
```

Comment:
What exactly are thos imported and exported functions. Would be nice if you could explain that a little more.

Reply:
This is something you can isolate using this tool. For more information about the imported or exported functions from a portable executable, you can for instance refer to [this article](https://blog.kowalczyk.info/articles/pefileformat.html).

Reply:
Doesn't sound like there's an easier way to describe it. Also, I don't assume this will print the whole function in binary, but just list the name or something about it. In this case we should use:
```suggestion
- List all imported functions:
```

Reply:
OK, excellent remark, thank you.